Another projekts/calculator/functions.py

```diff
@@ -0,0 +1,18 @@
+def mathematical_operation(choise):
+    first_number = int(input("Enter first number: "))
+    second_number = int(input("Enter second number: "))
+
+    if choise == '1':
+        result = first_number + second_number
+        print(f"The answer is {result}")
+    elif choise == '2':
+        result = first_number - second_number
+        print(f"The answer is {result}")
+    elif choise == '3':
+        result = first_number * second_number
+        print(f"The answer is {result}")
+    elif choise == '4':
+        result = first_number / second_number
+        print(f"The answer is {result}")
+    else:
+        print(f"Your value in not 1-4")
```

Comment:
Why does it happen again? (No Last empty line)
Please look at the File changes tab and re-check carefully all your changes in code before you ask for the code review.

Reply:
Missed it.
I wouldn't be able to compare it to the previous file because that file was created last time